h/presenters/annotation_searchindex.py

```diff
@@ -11,6 +11,9 @@ def __init__(self, annotation, request):
         self.request = request
 
     def asdict(self):
+        if self.annotation.deleted:
+            return self.deleted_annotation_body()
```

Comment:
`AnnotationSearchIndexPresenter.asdict()` is used to turn a `models.Annotation` object into a dict for indexing into Elasticsearch. This adds support for generating dicts for annotations that're marked as `deleted` in the DB